src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -934,14 +949,35 @@ table,
 	-webkit-appearance: none;
 	/* Removes default arrow from firefox */
 	text-indent: 0.01px;
-	text-overflow: "";
+}
+/* cancel stylistic differences in padding
```

Comment:
When does it happen? We don't use `appearance: none` ourselves after all.

Reply:
I think we effectively are, kind-of… I guess… at least with `appearance: none` it looks the same, IDK about backwards-compatibility with older versions of Firefox where it might’ve been different..?

In any case, what this is supposed to say is that Firefox has some (empirically, 4px) padding in an "unstyled" <select> (i.e. when the native arrow is removed) that Chrome doesn't have. I don't have any other browsers myself ATM to test / compare.

Admitted, given that `appearance: none` is **not** literally what’s used here, the comment could be clarified.

Reply:
I changed the wording a bit now.

I’m not sure about the pros and cons of using the “actual” `appearance: none` property, I haven’t researched too much what distinguishes this property from those `-moz-appearance` / `-webkit-appearance` ones; consequently I kept everything as-is with regards to these properties, just to be safe, e.g. since I haven’t tested other browsers, etc…